api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -28,6 +28,16 @@ import (
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
 func (in *Image) DeepCopyInto(out *Image) {
 	*out = *in
+	if in.Names != nil {
```

Comment:
why did this get updated?

Reply:
`make generate` hasn't been run in a while, and it got run as part of `make test`. The changes in #477 caused this, as it changed the definition of the `Image` type. We should probably implement a codegen check for the CI, similar to the manifest check we already have

Reply:
Just added that check to the CI for this PR